src/analysis/buildTypeWeights.ts

```diff
@@ -142,28 +169,28 @@ function parseObjectFields(
                                             variables[node.name.value] || arg.defaultValue
                                         );
                                     }
-                                    return multiplier * (selectionsCost + weight);
                                     // ? what else can get through here
+                                } else if (arg.defaultValue) {
+                                    // if there is no argument provided with the query, check the schema for a default
+                                    multiplier = Number(arg.defaultValue);
                                 }
-                                // if there is no argument provided with the query, check the schema for a default
-                                if (arg.defaultValue) {
-                                    return Number(arg.defaultValue) * (selectionsCost + weight);
-                                }
-
-                                // FIXME: The list is unbounded. Return the object weight for
-                                throw new Error(
-                                    `ERROR: buildTypeWeights: Unbouned list complexity not supported. Query results should be limited with ${KEYWORDS}`
-                                );
+                                // if there is no argument or default value, return 0 complexity
```

Comment:
The comment says return 0 but the code could returning something else

Reply:
This has been updated with a more descriptive comment

Reply:
@evanmcneely I don't see this update